profiles/coreos/base/make.defaults

```diff
@@ -94,3 +94,6 @@ PORTAGE_WORKDIR_MODE="0755"
 
 # Don't let GRUB whine about /boot
 DONT_MOUNT_BOOT=1
+
+# Use fast compression for binpkgs since we mostly compress then immediately uncompress them
```

Comment:
That is not true.  Binary packages are always uploaded and downloaded between workspaces in every CI build.  In addition, they are documented as being available to users in their developer images, containers, and SDKs, so it will affect some end-user download experience as well.

I disagree about compression speed being the right priority for binary packages, but not enough to block this if someone else wants to approve it.

Reply:
Hmm. It should be much faster locally (rebuilding things now). Maybe don't change make.defaults by default, but still update portage and document it in sdk-tips-and-tricks? If network bandwidth is an issue and we can figure out how to not break users, switching to xz might also be a good idea.

Reply:
I would be okay with changing the default to prioritize minimal file size, so xz would be okay.  (I believe Fedora still uses xz for RPM payload compression.)